tests/Database/DatabaseEloquentBelongsToManyTest.php

```diff
@@ -116,7 +116,7 @@ public function testModelsAreProperlyMatchedToParents()
         $this->assertEquals(2, $models[1]->foo[0]->pivot->user_id);
         $this->assertEquals(2, $models[1]->foo[1]->pivot->user_id);
         $this->assertEquals(2, count($models[1]->foo));
-        $this->assertEquals(0, count($models[2]->foo));
+        $this->assertEquals(0, count((array) $models[2]->foo));
```

Comment:
@laurencei I agree with your point and actually I don''t mind if you guys want to close this and support php7.2 only on 5.5. However IMO dropping the support just because ppl might upgrade their PHP version w.o reading the breaking changes also doesn't seems like the best solution for our problem.

Also I found funny the excuse given on the Backward Incompatible Changes section of the RFC:
> Environments that display warnings or convert them to more severe errors/exceptions would be affected, but this should just bring attention to a bug in the code.

I'm really surprised that there was no vote against this change.